include/modem/location.h

```diff
@@ -485,6 +485,34 @@ struct location_config {
 	 * location_config_defaults_set() function is called.
 	 */
 	enum location_req_mode mode;
+
+	/**
+	 * @brief Flag to request location back from the cloud.
+	 *
+	 * @details For cellular and Wi-Fi location methods, if this flag is set, the
+	 * cloud will return the current location, if known. The cloud default is true.
+	 */
+	bool do_reply;
+
+	/**
+	 * @brief Flag to control horizontal positioning error (HPE) for a location.
+	 *
+	 * @details For cellular and Wi-Fi location methods, if this flag is set, the
+	 * cloud will use a 95% confidence interval for the returned location and radius
+	 * than the default 68%. The cloud default is false.
```

Comment:
"...radius than" does not sound right. Maybe "...radius larger than the default 68%.", or something like that.

Reply:
I changed it to be "the cloud will use a 95% confidence interval for the returned location and radius instead of the default 68% confidence interval."